pkg/reconciler/service/service_test.go

```diff
@@ -174,12 +175,14 @@ func TestReconcile(t *testing.T) {
 		Key: "foo/byo-rev",
 		WantStatusUpdates: []clientgotesting.UpdateActionImpl{{
 			Object: DefaultService("byo-rev", "foo", WithNamedRevision,
-				WithInitSvcConditions, MarkRevisionNameTaken),
+				WithInitSvcConditions, WithServiceObservedGenFailure, MarkRevisionNameTaken,
```

Comment:
`WithServiceObservedGenFailure` should be dropped here since `MarkRevisionNameTaken` should fail the top level condition

Reply:
Thanks!